cfme/tests/infrastructure/test_resources.py

```diff
@@ -0,0 +1,36 @@
+from utils import testgen
+import pytest
+
+pytestmark = [pytest.mark.meta(server_roles='+smartproxy +smartstate')]
+
+
+def pytest_generate_tests(metafunc):
+    argnames, argvalues, idlist = testgen.infra_providers(
+        metafunc, required_fields=["ssh_credentials"])
+    testgen.parametrize(metafunc, argnames, argvalues, ids=idlist,
+                        scope="module")
+
+
+def get_integer_value(x):
+    return int(x.split(' ')[0])
+
+
+@pytest.mark.usefixtures("setup_provider_modscope")
+def test_number_of_cpu(provider, soft_assert):
+    soft_assert(get_integer_value(
```

Comment:
The test needs a comment, like eg. [here](https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/middleware/test_middleware_datasource.py#L19)